### PR TITLE
Adding IsNull, IsNotNull, IsEmpty, IsNotEmpty filters.

### PR DESCRIPTION
**Pull request by mehmetuken**

### CODE REVIEW

File: src/AutoFilterer/Attributes/OperatorComparisonAttribute.cs [hunk 1]
```diff
@@ -40,6 +41,10 @@ public override Expression BuildExpression(Expression expressionBody, PropertyIn
                 return Expression.LessThan(prop, param);
             case OperatorType.LessThanOrEqual:
                 return Expression.LessThanOrEqual(prop, param);
+            case OperatorType.IsNull:
+                return Expression.Equal( Expression.Property(expressionBody, targetProperty.Name), Expression.Constant(null));
+            case OperatorType.IsNotNull:
+                return Expression.NotEqual( Expression.Property(expressionBody, targetProperty.Name), Expression.Constant(null));
```
Comment (enisn) [inline src/AutoFilterer/Attributes/OperatorComparisonAttribute.cs]:
```suggestion
                return Expression.Equal(Expression.Property(expressionBody, targetProperty.Name), Expression.Constant(null));
            case OperatorType.IsNotNull:
                return Expression.NotEqual(Expression.Property(expressionBody, targetProperty.Name), Expression.Constant(null));
```

File: src/AutoFilterer/Attributes/OperatorComparisonAttribute.cs [hunk 2]
```diff
@@ -54,6 +59,10 @@ public override Expression BuildExpression(Expression expressionBody, PropertyIn
     public static OperatorComparisonAttribute LessThan { get; }
     public static OperatorComparisonAttribute LessThanOrEqual { get; }
 
+    public static OperatorComparisonAttribute IsNull { get; }
+
+    public static OperatorComparisonAttribute IsNotNull { get; }
+
```
Comment (enisn) [inline src/AutoFilterer/Attributes/OperatorComparisonAttribute.cs]:
```suggestion
    public static OperatorComparisonAttribute LessThanOrEqual { get; }
    public static OperatorComparisonAttribute IsNull { get; }
    public static OperatorComparisonAttribute IsNotNull { get; }

```

**Review by enisn — CHANGES_REQUESTED** [review]
Great job ✨🚀, thanks

I've just wanted some formatting changes to keep consistency in the project

**Comment by enisn** [comment]
Also, some of the tests are failing currently

Can you please update the following test according to the new logic?
https://github.com/enisn/AutoFilterer/blob/e47ce554a73056b8d5dfe1720d07092e6834bc18/tests/AutoFilterer.Tests/Types/OperatorQueryTests.cs#L36-L56

**Comment by mehmetuken** [comment]
> Also, some of the tests are failing currently


Not-Nullable types getting error. Adding the type is nullable check binary expressions.

**Review by enisn — APPROVED** [review]

**Comment by enisn** [comment]
I've just referenced the issue which is found by CodeFactor
https://github.com/enisn/AutoFilterer/issues/49

It's ok to merge now. I'll refactor it later. Using if chains is not a good solution, but it's ok for now.

**Pull request merged**